examples/graphics/material-physical-cubemap-rotation.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>PlayCanvas Physical Material</title>
```

Comment:
Rename to match the file name.
Do we need new very similar example or should we merge those two into a single example?

Reply:
I would probably agree here. I'm not sure this warrants a dedicated example to demonstrate a single property.

Reply:
hmm - I will merge it into one - the skybox rotation could be seen as adding interactivity to the example to better show how physical materials change with lighting.

Reply:
done - thanks